components/crud-web-apps/volumes/Dockerfile

```diff
@@ -7,7 +7,7 @@ COPY ./common/backend/ .
 RUN python3 setup.py bdist_wheel
 
 # --- Build the frontend kubeflow library ---
-FROM node:12 as frontend-kubeflow-lib
+FROM node:12-buster-slim as frontend-kubeflow-lib
```

Comment:
why do we use `buster-slim` in this case and not just `slim`? I see that you've removed the `buster` variant in the other images